public/styles/components/nav-primary/_nav-utility.scss

```diff
@@ -1,61 +1,80 @@
 // Utility Nav
 .bsds-nav-utility {
-  display: none;
-  border-bottom: $border-width-xxs solid $border-color-subtle;
-
   .bsds-nav {
-    justify-content: flex-end;
+    display: none;
+  }
 
-    .bsds-nav-item {
-      margin-top: 0;
+  .bsds-nav-logo {
+    display: flex;
+    margin: 0 auto;
+    padding: space(sm);
```

Comment:
These changes allow the secondary logo to show in the `bsds-nav-utility` container while keeping the utility nav hidden until it's supposed to show.